lab_02/readme.md

```diff
@@ -0,0 +1,11 @@
+# LAB-02 Maak een Twitter sniffer - kafka producer
+
+-   Zorg dat je een development Twitter account hebt
+-   neem de voorbeeldcode en pas deze aan zodat alle #trump berichten eruit gefilterd worden
```

Comment:
het is wel duidelijk wat je bedoelt, maar "eruit gefilterd" klinkt juist als het tegenovergestelde.